### PR TITLE
feat: adding citymapper

### PR DESCRIPTION
**Pull request by Kiruel**
Adding citymapper application to the launcher.

### CODE REVIEW

File: lib/map_launcher.dart [hunk 1]
```diff
@@ -101,13 +103,16 @@ class MapLauncher {
...
+    String address,
   }) async {
     final url = _getMapUrl(mapType, coords, title, description);
+
     final Map<String, String> args = {
       'mapType': _enumToString(mapType),
       'url': Uri.encodeFull(url),
       'title': title,
       'description': description,
+      'address': address,
```
Comment (mattermoran) [inline lib/map_launcher.dart:115]: I don't see where this property is being used

**Comment by mattermoran** [comment]
Hey @Kiruel It's looking good so far. I'll do a quick test and merge

**Comment by Kiruel** [comment]
@mattermoran Thanks for your quick review :)

**Comment by mattermoran** [comment]
@Kiruel no problem at all! Though it looks like citymapper is not supported in my current location so I cannot see what launching it looks like. You think you could make a short video or a gif of how it shows up on your phone when opening with this plugin?

**Comment by Kiruel** [comment]
@mattermoran You should use simulator of Android with the play store in it. And you can simulate the GPS to Paris for example and see what's going on. But yeah I can make a quick video to show you the result. (tested on iPhone XR, and OnePlus 5T and simulator Android).

**Comment by Kiruel** [comment]
Screen Recording 2020-06-02 at 03 04 05

**Comment by mattermoran** [comment]
It's usually problematic to make simulator work with the proxy so I didn't even try haha. Thanks for the vid. Looks good to me

**Comment by mattermoran** [comment]
Hey @Kiruel 
Version 0.5.0 with your changes should now be available on pub

**Pull request merged**